src/components/Card.js

```diff
@@ -19,9 +11,8 @@ export default class Card extends Component {
 
   constructor(properties) {
     super(properties);
-    this.state = {
-      childProps: sanitizeChildProps(properties)
-    };
+    const { style, ...childProps } = properties;
+    this.childProps = childProps;
```

Comment:
I would suggest and make it consistent between all Belle components and move child props out of the state.